<launch>

  <node pkg="line_detection" name="generic_linedetection" type="generic-linedetection.py" output="screen">
  </node>

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen" />
  
</launch>